### ISSUE TITLE
Plugin search got worse since v4.65.1

### ISSUE THREAD

**Issue by maxileith** [oznu/homebridge-config-ui-x#2288]
### Describe The Bug

There was an issue opened in my repository https://github.com/maxileith/homebridge-appletv-enhanced/issues/809. It seems like the the new search https://github.com/homebridge/homebridge-config-ui-x/pull/2280 is not returing the expected plugins when searching. As you can see from the issue that I have linked, the users search for "AppleTV Enhanced" which should clearly return the AppleTV Enhanced plugin, but it doesn't. No matter what I use as the search item, I cannot find the plugin.

I think we should revert the change and stick to official APIs from npmjs. What do you guys think?

### Homebridge UI Version

v4.56.1

### Homebridge Version

v1.8.5

### Node.js Version

22.12.0

### Operating System

Other (specify in description)

### Environment Info

Using Docker

### Raspberry Pi Model

None

**Comment by NorthernMan54** [comment]
@maxileith I would have to agree with you, #2287 and I had a user reach out to me directly supports this

@justjam2013 Thoughts ?

**Comment by justjam2013** [comment]
I haven't upgraded to v4.65.1 yet. However I know that for v5.0.0-beta.22 there is an issue with the search. The API was switched to what appeared to return better search results, but then discovered that some plugins are not being returned, even if searched by name.
I have opened a new PR that includes a roll back to the original API: [Reverted search URL and added filter to results](https://github.com/homebridge/homebridge-config-ui-x/pull/2286)
It may be that the changes to the search were released in v4.65.1. For now, my suggestion is to downgrade back to v4.65.0, while we reach a working solution. Alternatively, the plugins can be installed manually using `npn install`, but that doesn't address the search itself.
